app/(main)/league/[leagueId]/entry/[entryId]/week/Week.interface.ts

```diff
@@ -1,7 +1,19 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import { INFLTeam, ILeague } from '@/api/apiFunctions.interface';
+import React from 'react';
+import {
+  INFLTeam,
+  ILeague,
+  IWeeklyPicks,
+  IUser,
+} from '@/api/apiFunctions.interface';
+
+export interface IData {
```

Comment:
Make this more specific. IData is a generic name that could be for anything. Match the data structure. IWeekData is a much clearer name.